CODEOWNERS

```diff
@@ -0,0 +1,143 @@
+lib/vdsm/common/compat.py               @tinez @nirs
+lib/vdsm/common/concurrent.py           @nirs
+lib/vdsm/common/config.py.in            @tinez
+lib/vdsm/common/constants.py.in         @tinez
+lib/vdsm/common/conv.py                 @mz-pdm
+lib/vdsm/common/cpuarch.py              @mz-pdm
+lib/vdsm/common/define.py               @mz-pdm
+lib/vdsm/common/dsaversion.py.in        @tinez
+lib/vdsm/common/errors.py               @nirs
+lib/vdsm/common/eventfd.py              @tinez
+lib/vdsm/common/exception.py            @nirs @mz-pdm
+lib/vdsm/common/filecontrol.py          @tinez
+lib/vdsm/common/fileutils.py            @almusil @nirs @mz-pdm
+lib/vdsm/common/function.py             @almusil
+lib/vdsm/common/glob.py                 @nirs @vjuranek
+lib/vdsm/common/hooks.py                @tinez
+lib/vdsm/common/hostdev.py              @mz-pdm
+lib/vdsm/common/hostutils.py            @almusil
+lib/vdsm/common/libvirtconnection.py    @mz-pdm
+lib/vdsm/common/lockfile.py             @tinez @nirs
+lib/vdsm/common/logutils.py             @nirs @mz-pdm @tinez
+lib/vdsm/common/nbdutils.py             @nirs
+lib/vdsm/common/osutils.py              @tinez
+lib/vdsm/common/panic.py                @nirs @tinez
+lib/vdsm/common/password.py             @nirs @mz-pdm @tinez
+lib/vdsm/common/pki.py                  @tinez
+lib/vdsm/common/proc.py                 @almusil
+lib/vdsm/common/properties.py           @nirs @vjuranek
+lib/vdsm/common/pthread.py              @mz-pdm @nirs @tinez
+lib/vdsm/common/response.py             @mz-pdm
+lib/vdsm/common/sigutils.py             @tinez @nirs
+lib/vdsm/common/supervdsm.py            @tinez @nirs
+lib/vdsm/common/systemctl.py            @nirs @vjuranek
+lib/vdsm/common/systemd.py              @nirs @vjuranek
+lib/vdsm/common/threadlocal.py          @tinez @nirs
+lib/vdsm/common/time.py                 @tinez @nirs
+lib/vdsm/common/udevadm.py              @nirs @vjuranek
+lib/vdsm/common/units.py                @nirs @vjuranek
+lib/vdsm/common/unixrpc.py              @mz-pdm
+lib/vdsm/common/validate.py             @mz-pdm
+lib/vdsm/common/xmlutils.py             @mz-pdm
+
+# Helpers
+helpers/fallocate                       @nirs @vjuranek
+helpers/kvm2ovirt                       @mz-pdm
+
+# Containers
+docker/                                 @nirs @almusil @tinez
+
+# Init
+init/                                   @tinez
+
+# Static
+static/                                 @tinez
+static/etc/libvirt                      @mz-pdm
+static/etc/modprobe.d                   @almusil
+static/etc/modules-load.d               @almusil
+static/etc/NetworkManager               @almusil
+static/etc/ovirt-imageio                @nirs @vjuranek
+static/etc/sudoers.d                    @tinez @nirs @vjuranek
+static/etc/vdsm/mom.d/                  @mz-pdm
+
+# Hooks
+vdsm_hooks/                             @tinez @mz-pdm @almusil
+
+# Documentation owners - maintainer needed
```

Comment:
Let's put all of us on it?

Reply:
Since we did not configure it, I think the default rule will match it:
https://github.com/oVirt/vdsm/pull/28/commits/96320682a8b41e440fbbb8ca624fd73efa95bda0#diff-fcf14c4b7b34fe7a11916195871ae66a59be87a395f28db73e345ebdc828085bR5

Reply:
But I hope we can get someone from the doc team to review changes here. @emarcusRH what do you think?

Reply:
Thanks @emarcusRH!

I can post a patch adding you as owner of README.md and doc/ directory, or maybe I should add
@oVirt/ovirt-documentation as owner?

Reply:
I guess that the @oVirt/ovirt-documentation  tag is fine - that's how our ovirt-site PRs are tagged - it is automatic whenever we open a new PR